Dysplasia.html

```diff
@@ -47,12 +48,12 @@ <h2 class="text-center">Dysplasia Form v 0.01</h2>
         <select class="form-control" id="patient.sex" name="patient.sex" disabled="disabled">
             <option value="">...</option>
             <option value="M">Male</option>
-            <option value="F">Female</option>
+            <option value="F" selected>Female</option>
```

Comment:
Do we want to select the female all the time or this shouldn't be here @mssavai?